testing/adios2/gpu-backend/TestBPWriteReadKokkos.cpp

```diff
@@ -367,8 +367,9 @@ bool compareSelection2D(
             if (b(x, y) != a(start_0 + x, start_1 + y))
             {
                 lmatch++;
-                Kokkos::printf("   Non-match at pos = (%d %d) : input = %f : output = %f\n", x, y,
-                               a(start_0 + x, start_1 + y), b(x, y));
+                // Add lines back when using Kokkos 4.2
+                // Kokkos::printf("   Non-match at pos = (%d %d) : input = %f : output = %f\n", x,
+                //               y, a(start_0 + x, start_1 + y), b(x, y));
```

Comment:
```suggestion
```

Reply:
I usually dislike commenting code. I left it here just because the c++ version of this testing is printing the mismatches and I wanted to be consistent. But I am also fine making a note to reintroduce this printf when we decide to move the min Kokkos version to 4.2